app/forms/gpo_verify_form.rb

```diff
@@ -43,7 +43,7 @@ def submit
         letter_count: letter_count,
         attempts: attempts,
         pii_like_keypaths: [[:errors, :otp], [:error_details, :otp]],
-        pending_in_person_enrollment: pending_profile&.pending_in_person_enrollment?,
+        pending_in_person_enrollment: has_pending_in_person_enrollment?,
```

Comment:
Given that the previous behavior didn't check for whether the enrollment was in `:pending` status, we should check how this changes the analytics data and what may currently rely on this field.

I suspect that this field may log `false` where it previously logged `true`.

Reply:
It's true that previous behavior didn't check for the enrollment's status, but I think the expectation, given its name, is that it should be true if the associated enrollment is pending and false otherwise. And, I think it should log `true` in most cases here, because we've scheduled the enrollment on line 26 above.